uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -443,6 +512,16 @@ define([
       }
 
       /**
+       * Converts given date to javascript date as expected by uib-datepicker
+       *
+       * @param {Date} date
+       * @return {Date} Javascript date
```

Comment:
The `@param` and `@return` types can't be right? From the documentation it seems that you are passing a `Date` object and getting a `Date` object as the output, so what's the point of the function?